### ISSUE TITLE
Add camptocamp/puppet-systemd

### ISSUE THREAD

**Issue by dhoppe** [deric/puppet-zookeeper#147]
The following module should be used to manage Systemd unit files and trigger the reload if something changes:
* https://github.com/camptocamp/puppet-systemd

**Comment by Fabian1976** [comment]
No, do not. The camptocamp module is deprecated and no longer maintained (https://forge.puppet.com/modules/camptocamp/systemd)

Please use puppet/systemd (https://forge.puppet.com/modules/puppet/systemd)
